Extensions/StyleCop/StyleCopTask/task.json

```diff
@@ -48,11 +48,21 @@
       "label": "Max. Violation Count",
       "defaultValue": "1000",
       "required": true,
-      "helpMarkDown": "Maximum violations before analysis stops",
+      "helpMarkDown": "Maximum project violations before analysis stops",
       "groupName":"advanced"
     }
     ,
     {
+      "name": "allowableViolationCount",
+      "type": "string",
+      "label": "Allowable Violation Count",
+      "defaultValue": "0",
+      "required": true,
+      "helpMarkDown": "Number of violations allowed before the task fails",
+      "groupName": "advanced"
+    }
+    ,
```

Comment:
The mix of styles already existed in the master, not sure why and was unsure what to do regarding it. I would agree though I prefer the comma on the line with the brace. If you're happy I'll go ahead and update the whole file to follow that style.